components/sup/src/manager/mod.rs

```diff
@@ -48,16 +49,16 @@ use serde;
 use serde_json;
 use time::{self, Timespec, Duration as TimeDuration};
 
-pub use manager::service::{Service, ServiceConfig, ServiceSpec, UpdateStrategy, Topology};
-use self::service::{DesiredState, StartStyle};
+pub use self::service::{Service, ServiceSpec, UpdateStrategy, Topology};
```

Comment:
The way I've organized these are

* Imports from std
* Public imports from extern
* Imports from extern
* Public imports from our own crate
* Imports from our own crate

I chose this since the rest of the code is formatted (or should be... we really need to commit some rules to the wiki!)

* Imports
* Public Const
* Private Const
* Public Static
* Private Static
* Public Trait
* Private Trait
* Public Struct
* Private Struct
* Public Function
* Private Function

We've got a few other small things in there like recently organizing members of the struct into public/private sections and organization of functions within the struct itself.

If you think we can come up with something more readable then let's go ahead and do it